.gitignore

```diff
@@ -1,10 +1,40 @@
-# System files
-.DS_Store
-Thumbs.db
-
-# Parcel-bundler cache
-.cache/
-
-# Temp files
-tmp/
-temp/
```

Comment:
Any particular reason you removed these? Are we sure no dependencies will create files of these types?

Reply:
Dunno I will remove
(Actually think it was from another PR)